src/Symfony/Component/Validator/Constraints/CallbackValidator.php

```diff
@@ -44,8 +44,11 @@ public function validate($object, Constraint $constraint)
             }
 
             \call_user_func($method, $object, $this->context, $constraint->payload);
+        } elseif (\method_exists($method, '__invoke')) {
+            $validator = new $method();
```

Comment:
what if `$method` is an invokable object? We need to check it before building a new instance, no?

Reply:
when the ```__invoke``` method is exist, we imagine the object is callable, and we need to create an object before because the ```__invoke``` only available in object

Reply:
@yceruto: in validator, you only pass the string of class name or a method name of object (entity), you can't passing object inside of validator (annotation) except it self (entity)

Reply:
Actually you can in some ways, using PHP. But that's probably discouraged as the constraint needs to be serializable to be cached, so could be an issue/not work as expected. A custom constraint & validator should be preferred.

Reply:
@ogizanagi : can you tell how to fix?